tests/geometry/lr_cost_test.py

```diff
@@ -72,6 +74,28 @@ def test_conversion_pointcloud(
             rtol=1e-4
         )
 
+  def test_conversion_pointcloud_dotp(
+      self,
+      rng: jax.Array,
+  ):
+    """Test conversion from PointCloud to LRCGeometry."""
+    n, m, d = 17, 11, 3
+    rngs = jax.random.split(rng, 3)
+    x = jax.random.normal(rngs[0], (n, d)) + .1
+    y = jax.random.normal(rngs[1], (m, d))
+
+    geom = pointcloud.PointCloud(x, y, cost_fn=costs.Dotp())
+    geom_lr = geom.to_LRCGeometry()
```

Comment:
Please add `isinstance(geom_lr, low_rank.LRCGeometry)` just to be sure.

Reply:
good catch